orthoinference/runOrthoinference.sh

```diff
@@ -12,7 +12,7 @@ allSpecies=(pfal spom scer ddis cele sscr btau cfam mmus rnor ggal xtro drer dme
 for species in "${allSpecies[@]}"
 do
 	echo "java -jar target/orthoinference-0.0.1-SNAPSHOT-jar-with-dependencies.jar $species > orthoinference_$species.out";
-	java -jar target/orthoinference-0.0.1-SNAPSHOT-jar-with-dependencies.jar $species > orthoinference.out;
+	java -jar target/orthoinference-0.0.1-SNAPSHOT-jar-with-dependencies.jar $species > orthoinference_$species.out;
```

Comment:
I would assume at some point the version number of the Java application will change, and there will need to be a variable in this script that contains the version number and it gets used as 
```bash
java -jar target/orthoinference-$ORTHOINFERENCE_VERSION-jar-with-dependencies.jar $species > orthoinference_$species.out;
```

Reply:
Would it make sense to try and get the version number from the jar file in the `target` folder?

Reply:
Me, I'd just have a variable in the script and ensure that it's updated every time this project is released. It *could* be automated, if you want, probably by getting maven to to a find-and-replace on a token in the shell script.

Reply:
But more importantly, update the version number in the pom file if this is no longer a snapshot. If it's ready to be used in production, you can set the version number to 1.0.0.